functions/controllers/kaiAIController.js

```diff
@@ -266,18 +269,21 @@ app.post('/api/tool/', (req, res) => {
 });
 
 /**
- * This creates a chat session for a user.
+ * Creates a chat session for a user.
  * If the chat session already exists, it will return the existing chat session.
  * Otherwise, it will create a new chat session and send the first message.
  *
+ * @async
+ * @function createChatSession
  * @param {Object} props - The properties passed to the function.
- * @param {Object} props.data - The data object containing the user, challenge, message, and botType.
+ * @param {Object} props.data - The data object containing the user, message, and type.
```

Comment:
The previous doc was more clear than this. Please put it back.

Reply:
modified as your suggestions